### ISSUE TITLE
No demo and no styles

### ISSUE THREAD

**Issue by bjodasso** [Venturocket/angular-slider#28]
You need to add an example/demo to the page and also...are there no default styles? do you expect everyone to style the whole thing on their own?

**Comment by bjodasso** [comment]
Ok I found the example html file, should really have this linked on the site... still wondering about the styles, would at least be nice to have bootstrapish styles.

**Comment by drgould** [comment]
I'm in the process of putting together a more complete demo than what's in the example.html file. I'll include some default styles along with it.

**Comment by bjodasso** [comment]
Great, using the styles from the example I was just running into issues with not being able to click or grab one of the handles...might not have been a style thing, but who knows.

**Comment by porjo** [comment]
Lack of styles would explain why nothing is displaying when I test! README should be updated to explain this (until default styles arrive).

**Comment by cranesandcaff** [comment]
Agree with porjo, the lack of styles is confusing and at the least should be indicated.

**Comment by joscha** [comment]
:+1:

**Comment by Tyvain** [comment]
Agree. And css in the demo is very ugly.

**Comment by AritzBi** [comment]
Where is the demo with the css files? I can't find it...

**Comment by devsathish** [comment]
Pls add a demo link. Atleast, it gives confidence that this library works.
I tried adding it to jsfiddle: http://jsfiddle.net/kk0td02z/
Seems something is broken.

**Comment by jlyonsmith** [comment]
+1 to adding some default styles!  I thought the library was broken and was about to drop it!  At least add a link to the above JSFiddle/Plunker demos.  Thank you for all the hard work on this directive!

**Comment by Shawful** [comment]
I agree that there should be a working jsfiddle.  Thanks devsathish for posting your attempt.

**Comment by jprince** [comment]
Has anyone developed some basic styles that they're willing to share?

**Comment by jprince** [comment]
Answered my own question ^^^. For those interested there are some styles linked to (potentially duplicative) issue https://github.com/Venturocket/angular-slider/issues/51

**Comment by jeffcai** [comment]
still no plan to provide demo and css?

**Comment by mojjj** [comment]
bump

**Comment by pain-x01** [comment]
So, for first we are must set next styles for holder (parent block, or slider ):
position:relative;
display: block;
width:100% (or fixed value)

and for .pointer  set width, height, cursor poiner, background

hope i'ts will help

**Comment by jprince** [comment]
Since people are apparently too lazy to click thru to the issue referenced in my comment above... original styles: https://github.com/prajwalkman/angular-slider/blob/master/angular-slider.css
